PURL-TYPES.rst

```diff
@@ -59,6 +59,24 @@ apk
       pkg:apk/alpine/curl@7.83.0-r0?arch=x86
       pkg:apk/alpine/apk@2.12.9-r3?arch=x86
 
+bazel
+-----
+``bazel`` for Bazel modules:
+
+- The default repository ("registry") is ``https://bcr.bazel.build``, the
```

Comment:
This forwards to https://registry.bazel.build/. So should the URL be changed here?

Reply:
https://bcr.bazel.build is indeed the default registry (see https://github.com/bazelbuild/bazel/blob/a298190ff22435f32dc3a3ddc6507ddc72e0e503/src/main/java/com/google/devtools/build/lib/bazel/BazelRepositoryModule.java#L139-L140) whereas https://registry.bazel.build offers a UI for the hosted modules. We recently added redirects from bcr.bazel.build to registry.bazel.build for those requests that a registry consumer wouldn't make.

For example, https://bcr.bazel.build/modules/rules_go/0.50.1/MODULE.bazel is what Bazel would download and doesn't redirect, but https://bcr.bazel.build/modules/rules_go/0.50.1 redirects to https://registry.bazel.build/modules/rules_go/0.50.1 as a convenience for humans.